README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
+here are some things you will need to know:
+* The default port for the Minecraft server is: `25565`.
+* If you wish to run multiple world servers using this script, you will
+  want to open a range of ports (for example `25565 - 25575`).
 
-### Mapping Software
+See the [iptables.rules](iptables.rules)
+file for a very basic set of rules that you can use with the Iptables firewall.
+
+### Mapping software (optional)
 The script uses the [Minecraft Overviewer](http://overviewer.org) mapping
 software to generate maps of your worlds.  Minecraft Overviewer is a
 command-line tool for rendering high-resolution maps of Minecraft worlds. It
 generates a set of static html and image files and uses the Google Maps API to
 display a nice interactive map.
 
-You can [download](http://overviewer.org/downloads) premade binaries for
+If you wish to use the mapping software, you can [download]
+(http://overviewer.org/downloads) premade binaries for
 supported systems, or build your own binary from source if needed.
 
 Repositories for automatic installation are also available:
 * [Debian/Ubuntu](http://overviewer.org/debian/info)
 * [RHEL/CentOS/Fedora](http://overviewer.org/rpms/info)
 
-### Firewall / NAT
-If you have a firewall installed on your computer, or a router using NAT
-installed in your network, you will need to route some ports to your server.
-Instructions on how to accomplish this are beyond the scope of this post, but
-here are some things you will need to know:
-* The default port for the Minecraft server is: `25565`.
-* If you wish to run multiple world servers using this script, you will
-  want to open a range of ports (for example `25565 - 25575`).
+## Installation
+### Downloading the script
+The easiest way to download the script is to make a clone of the [git
+repository](https://github.com/sandain/MinecraftServerControlScript.git). 
+You must have git installed first. To install git:
 
-See the [iptables.rules](iptables.rules)
-file for a very basic set of rules that you can use with the Iptables firewall.
+        sudo apt-get install git
+        
+Then:
+
+        git clone https://github.com/sandain/MinecraftServerControlScript.git
+        
+Note that it will be downloaded into the current directory which you are working
+ in. 
+
+##### Other ways to download
+
+* Get the latest stable [release]
+(https://github.com/sandain/MinecraftServerControlScript/releases).
+
+* Get the development version as a [zip file]
+(https://github.com/sandain/MinecraftServerControlScript/archive/master.zip):
+
+        wget https://github.com/sandain/MinecraftServerControlScript/archive/master.zip
+
+
+
+### Configuration
+
+Navigate to the `MinecraftServerControlScript` directory that you just 
+downloaded. 
+Configuration can be done with the included Makefile in Debian and
+Ubuntu like environments by running:
+
+        sudo make install
+Then, type
+
+        chmod -R u+w /opt/mscs
+        chown -R minecraft:minecraft /opt/mscs
```

Comment:
Hmmm... could it be possible it would function differently on a different computer? When I ran `sudo make install` I had to `chmod` it in order for it to create the folders.

I'll create and add it to a troubleshooting section, though